install/k8s/contiv/contiv.yaml

```diff
@@ -0,0 +1,257 @@
+---
+# This ConfigMap is used to configure a self-hosted Contiv installation.
```

Comment:
Yes, consul can be used with Contiv. It would be nice if the user has an option to just specify a state store (etcd or consul) instead of installing etcd.